src/game/Object.cpp

```diff
@@ -517,8 +517,18 @@ void Object::BuildValuesUpdate(uint8 updatetype, ByteBuffer* data, UpdateMask* u
                                 break;
                         }
                     }
-                    else
-                        *data << uint32(0);                 // disable quest object
+                    else    // flag (no interact) quest object when player not eligible for quest
```

Comment:
as requested, move comment up to else as it applies to the whole else statement.